### ISSUE TITLE
Hw 1 grade and comment

### ISSUE THREAD

**Issue by anglipku** [Uchicago-Stat-Comp-37810/assignment-1-xiaomengSSA#1]
Grade: 8.5 / 10
Comment:
Good work on this homework, but there are a few things to pay attention (see below).  Also if you have collaborated with others, please mention this at the beginning of homework.

4.1.1.2: need to show graphically that the ratio converges.
4.4.1.2: Where is the output of this function ? Also, how are f, g, fx, fy, gx, gy, x0, y0 defined ?

**Comment by xiaomengSSA** [comment]
Good afternoon TA Ang,
This is Xiaomeng from SSA and I collaborated with Lu Yang in this assignment. Thank you so much for your comment. Since I have an internship on Friday and is it possible to make an appointment with you on Tuesday afternoon or Thursday afternoon to discuss some exercises in our lectures and  possible suggestions to improve my study in this course. Or, if it is possible to send you emails to ask specific questions about our assignment? Since I have no programming experiences, and I basically took some online R courses in coursera and mooc, the lectures in terms of MCMC, Gibbs sampling still are difficult for me and I do want to find some suggestions on it! Many thanks!

**Comment by anglipku** [comment]
Hi Xiaomeng:

I understand that many students in the class are not familiar with sampling
methods and Bayesian statistics, and will take this into concern when
grading.  Don't worry too much about Hw 2.  This week I can only meet at
Friday.  Perhaps you can discuss with Lu after she attend the office hour,
and find a time to discuss with me later.

Best wishes,
Ang Li

**Comment by xiaomengSSA** [comment]
Thank you so much and I am taking an online probability system course and the fundamental ideas about probability helps me a little bit. There is another issue and I just want to seek some possible answers. Since this course on the register system does not display the exact period and the concurrent requirement of "37820". I failed to choose 37820 and want to know if students are required to choose both historically? Would you please give me some suggestions on it? Again, many thanks.

**Comment by KBurbank** [comment]
No, you are not required to take both courses at the same time. However, it may still be possible to register for the second course. You should email Mei Wang to see if it’s possible!
